test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1

```diff
@@ -195,30 +195,22 @@ Describe "Deserializing corrupted Cim classes should not instantiate non-Cim typ
 
         # Only run on Windows platform.
         # Ensure calc.exe is avaiable for test.
-        if ( !$IsWindows -or ((Get-Command calc.exe 2>$null) -eq $null) )
-        {
-            $orginalDefaultParameters = $PSDefaultParameterValues.Clone()
-            $PSDefaultParameterValues["it:skip"] = $true
-        }
-        else
+        $shouldRunTest = $IsWindows -or ((Get-Command calc.exe -ea SilentlyContinue) -ne $null)
+        $skipNotWindows = ! $shouldRunTest
```

Comment:
I personally like this a lot better.  But with the previous pattern the issue was variable scoping.  It should have been $global:PSDefaultParameters, correct?